webrtc-stats.html

```diff
@@ -2049,6 +2050,28 @@ <h2>
                   MediaStreamTrack, if <code>type</code> is <code>"track"</code>).
                 </p>
               </dd>
+              <dt>
+                <dfn><code>hugeFramesSent</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of huge frames sent by this RTCRtpSender (or for this
+                  MediaStreamTrack, if <code>type</code> is <code>"track"</code>). Huge frames, by
+                  definition, are frames with encoded size at least 2.5 times the typical frame size,
```

Comment:
are frames that have an encoded size of at least 2.5 times the average size of the frames. The average size of the frames is defined as the target bitrate per second divided by the target frame rate at the time when the frame was encoded.